src/main/java/org/jenkinsci/plugins/periodicbackup/FullBackup.java

```diff
@@ -68,9 +69,10 @@ public FullBackup(@CheckForNull String excludesString) {
      * @param excludesString Optional list of directories to be excluded
      * @param baseDir Base directory
      */
-    FullBackup(@CheckForNull String excludesString, @Nonnull File baseDir) {
+    FullBackup(@CheckForNull String excludesString, @CheckForNull boolean followSymbolicLinks, @Nonnull File baseDir) {
```

Comment:
CheckForNull redundant, primitives cannot be null.

Reply:
removed it!